hikari/impl/stateful_cache.py

```diff
@@ -863,23 +893,23 @@ def delete_emoji(self, emoji_id: snowflake.Snowflake, /) -> typing.Optional[emoj
     def get_emoji(self, emoji_id: snowflake.Snowflake, /) -> typing.Optional[emojis.KnownCustomEmoji]:
         return self._build_emoji(self._emoji_entries[emoji_id]) if emoji_id in self._emoji_entries else None
 
-    def _get_emojis_view(
+    def _get_emojis_view(  # TODO: split out?
```

Comment:
Split out how?

If I had to take over from you, it should tell me what exactly the intent is :+1: